### ISSUE TITLE
Add lang attribute to html

### ISSUE THREAD

**Issue by ALacker** [activeadmin/activeadmin#5291]
WCAG 2.0 A compliance (for a11y) mandates a lang attribute on the html tag. https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=311#meaning-doc-lang-id

There is currently no lang attribute, but it has been requested for other reasons as well, such as RTL language support (https://github.com/activeadmin/activeadmin/issues/362#issuecomment-68255422)

**Comment by Fivell** [comment]
closed by #5292

**Comment by pockata-kt** [comment]
Why has this never been included in a release?
